### DISCUSSION TITLE
No funciona en dispositivos Poco x3 GT

### DISCUSSION THREAD

**Discussion by lex4nd3r** [skyline-emu/skyline#1859] [General]
Esta app se congela en dispositivos mediatek dimensity. Por favor arreglen eso.

**Comment by Allatel** [comment]
This app does not freeze on mediatek dimensity devices. Please fix that.
That what Google translate did. 
Why do you need fix for hat?

**Reply by Dekuzearou** [reply]
So you want the app to freeze?

**Reply by lex4nd3r** [reply]
Quizas entendieron mal por que use google translate. Quize decir que este emulador es obsoleto en CPU MEDIATEK DIMENSITY.

**Reply by Raziel19935** [reply]
mierdaTek trabaja con mali, compra otro teléfono..
